src/rpc/blockchain.cpp

```diff
@@ -1107,7 +1107,7 @@ static RPCHelpMan verifychain()
 
     Chainstate& active_chainstate = chainman.ActiveChainstate();
     return CVerifyDB().VerifyDB(
-        active_chainstate, chainman.GetParams().GetConsensus(), active_chainstate.CoinsTip(), check_level, check_depth);
+               active_chainstate, chainman.GetParams().GetConsensus(), active_chainstate.CoinsTip(), check_level, check_depth) == VerifyDBResult::SUCCESS;
```

Comment:
nit: not sure about those extra spaces there; could you please double check what's mentioned in the [developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md) about [AlignAfterOpenBracket](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) style option?

Reply:
I ran the Clang formatter script over the commits (as described in `/contrib/devtools/README.txt`) and I'm pretty sure it added the space for me.